packages/management/addons-manager/1.4.0/bundle/config/overlays/overlay-addons-manager.yaml

```diff
@@ -0,0 +1,57 @@
+#@ load("@ytt:data", "data")
```

Comment:
we don't need to keep the packages versioned in the repo, we version the packages at build time, so version directory can  be removed

Reply:
removed version directory